tasks/secrets_protection.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Add Confluent CLI to path
+  shell: "export PATH=/usr/local/bin/bin:${PATH};"
+
+- name: Create Passphrase File
+  shell: "openssl rand -base64 14 > passphrase.txt"
+
+- name: Generate Master Encryption Key
+  shell: "confluent secret master-key generate --local-secrets-file security.properties \
+           --passphrase @passphrase.txt | awk '/Master/{print $5}'"
+  register: masterkey
```

Comment:
the goal would be not to generate the master key each time, but do it once and save that key as a variable for all hosts (or pass in as a variable)